sherpa/astro/ui/utils.py

```diff
@@ -8635,18 +8637,25 @@ def fake_pha(self, id, arf, rmf, exposure, backscal=None, areascal=None,
            dataset identifier is used, rather than creating a
            completely new dataset.
 
+        .. versionchanged:: 4.13.2
```

Comment:
```suggestion
        .. versionchanged:: 4.14.0
```

I don't believe we are having a 4.13.2 release.

Reply:
And we might not want the changelog in the docstring anyway...